extra/release_tool.py

```diff
@@ -584,12 +587,19 @@ def do_version_of(args):
         print("Unrecognized repository: %s" % args.version_of)
         sys.exit(1)
 
-    yml_component = comp.yml_components()[0]
-
     assert args.version_type in ["docker", "git"], (
         "%s is not a valid name type!" % args.version_type
     )
 
+    if comp.type in ["docker_image", "docker_container"] and args.version_type == "git":
+        print(
+            "Unsuported %s %s for --version-type git. Use --version-type docker instead"
+            % (comp.type, args.version_of)
+        )
+        sys.exit(1)
```

Comment:
I disagree with this check. All containers and images have a Git repository, so there should never be a reason not to return a result for `version_type == "git"`. See previous comment also.

Reply:
Related to the above. Let's keep the discussion there: if we find a good solution I will happily remove the restriction.